Internal Service Error

It looks like the tables are not created with docker-compose quickstart. 

```shell
todolist     | [2020-02-01 18:59:33 +0000] [8] [ERROR] Error handling request /todolist/new/
todolist     | Traceback (most recent call last):
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1246, in _execute_context
todolist     |     cursor, statement, parameters, context
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
todolist     |     cursor.execute(statement, parameters)
todolist     | sqlite3.OperationalError: no such table: todolist
todolist     | 
todolist     | The above exception was the direct cause of the following exception:
todolist     | 
todolist     | Traceback (most recent call last):
todolist     |   File "/usr/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 134, in handle
todolist     |     self.handle_request(listener, req, client, addr)
todolist     |   File "/usr/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 175, in handle_request
todolist     |     respiter = self.wsgi(environ, resp.start_response)
todolist     |   File "/usr/lib/python3.7/site-packages/flask/app.py", line 2463, in __call__
todolist     |     return self.wsgi_app(environ, start_response)
todolist     |   File "/usr/lib/python3.7/site-packages/flask/app.py", line 2449, in wsgi_app
todolist     |     response = self.handle_exception(e)
todolist     |   File "/usr/lib/python3.7/site-packages/flask/app.py", line 1866, in handle_exception
todolist     |     reraise(exc_type, exc_value, tb)
todolist     |   File "/usr/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
todolist     |     raise value
todolist     |   File "/usr/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
todolist     |     response = self.full_dispatch_request()
todolist     |   File "/usr/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
todolist     |     rv = self.handle_user_exception(e)
todolist     |   File "/usr/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
todolist     |     reraise(exc_type, exc_value, tb)
todolist     |   File "/usr/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
todolist     |     raise value
todolist     |   File "/usr/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
todolist     |     rv = self.dispatch_request()
todolist     |   File "/usr/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
todolist     |     return self.view_functions[rule.endpoint](**req.view_args)
todolist     |   File "/code/app/main/views.py", line 46, in new_todolist
todolist     |     todolist = TodoList(creator=_get_user()).save()
todolist     |   File "/code/app/models.py", line 45, in save
todolist     |     self.__commit()
todolist     |   File "/code/app/models.py", line 33, in __commit
todolist     |     db.session.commit()
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/scoping.py", line 162, in do
todolist     |     return getattr(self.registry(), name)(*args, **kwargs)
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 1036, in commit
todolist     |     self.transaction.commit()
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 503, in commit
todolist     |     self._prepare_impl()
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 482, in _prepare_impl
todolist     |     self.session.flush()
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2479, in flush
todolist     |     self._flush(objects)
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2617, in _flush
todolist     |     transaction.rollback(_capture_exception=True)
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 68, in __exit__
todolist     |     compat.reraise(exc_type, exc_value, exc_tb)
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 153, in reraise
todolist     |     raise value
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2577, in _flush
todolist     |     flush_context.execute()
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 422, in execute
todolist     |     rec.execute(self)
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 589, in execute
todolist     |     uow,
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 245, in save_obj
todolist     |     insert,
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 1137, in _emit_insert_statements
todolist     |     statement, params
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 982, in execute
todolist     |     return meth(self, multiparams, params)
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 293, in _execute_on_connection
todolist     |     return connection._execute_clauseelement(self, multiparams, params)
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1101, in _execute_clauseelement
todolist     |     distilled_params,
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1250, in _execute_context
todolist     |     e, statement, parameters, cursor, context
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1476, in _handle_dbapi_exception
todolist     |     util.raise_from_cause(sqlalchemy_exception, exc_info)
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 398, in raise_from_cause
todolist     |     reraise(type(exception), exception, tb=exc_tb, cause=cause)
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 152, in reraise
todolist     |     raise value.with_traceback(tb)
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1246, in _execute_context
todolist     |     cursor, statement, parameters, context
todolist     |   File "/usr/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
todolist     |     cursor.execute(statement, parameters)
todolist     | sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: todolist
todolist     | [SQL: INSERT INTO todolist (title, created_at, creator) VALUES (?, ?, ?)]
todolist     | [parameters: ('untitled', '2020-02-01 18:59:33.031390', None)]
todolist     | (Background on this error at: http://sqlalche.me/e/e3q8)
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,21 +1,10 @@
-version: '2'
+version: "2"
 services:
   todolist:
     container_name: todolist
     image: todolist
-    # restart: always
+    build: .
     env_file: .env
-    command: gunicorn todolist:app -w 2 -b :8000
-    volumes:
-      - .:/code
+    command: sh -c "flask db upgrade && gunicorn todolist:app -w 2 -b :8000"
     ports:
       - "8000:8000"
-
-  migration:
-    container_name: migration
-    build: .
-    image: todolist
-    env_file: .env
-    command: flask db upgrade
-    depends_on:
-      - todolist

